actor-to-actor/ping-interface/src/generated.rs

```diff
@@ -0,0 +1,105 @@
+extern crate rmp_serde as rmps;
+use rmps::{Deserializer, Serializer};
+use serde::{Deserialize, Serialize};
+use std::io::Cursor;
+
+extern crate log;
+extern crate wapc_guest as guest;
+use guest::prelude::*;
+
+use lazy_static::lazy_static;
+use std::sync::RwLock;
+
+pub struct Host {
+    binding: String,
+}
+
+impl Default for Host {
+    fn default() -> Self {
+        Host {
+            binding: "default".to_string(),
+        }
+    }
+}
+
+/// Creates a named host binding for the key-value store capability
+pub fn host(binding: &str) -> Host {
+    Host {
+        binding: binding.to_string(),
+    }
+}
+
+/// Creates the default host binding for the key-value store capability
+pub fn default() -> Host {
+    Host::default()
+}
```

Comment:
Do these two functions get generated with the help text `key-value store capability`? Either way here I think it should be the "ping" capability, but we should open an issue with the codegen if that's hard coded

Reply:
Yeah it looks like that stuff is part of the rust template and isn't correctly pulling in the widl comments. I've fixed it manually but we should make sure there's issue coverage on the wapc rust generator.